config/config.go

```diff
@@ -273,6 +274,12 @@ func (s *command) Execute(descriptors ...definition.Descriptor) error {
 
 // ExecuteWithConfig runs nirvana server from a custom config.
 func (s *command) ExecuteWithConfig(cfg *nirvana.Config) error {
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+	// https://github.com/kubernetes/kubernetes/issues/17162#issuecomment-225596212
+	if err := flag.CommandLine.Parse([]string{}); err != nil {
```

Comment:
These codes should not be here. It's a special case for your project.

Reply:
reasonable, I'll close this